.github/workflows/e2e-ci.yml

```
@@ -45,6 +45,9 @@ jobs:
       - name: Make jest available
         working-directory: ${{env.working-directory}}
         run: cp ../../node_modules/.bin/jest node_modules/.bin/
+      - name: Check E2E wallet balance
+        working-directory: ${{env.working-directory}}/e2e/scripts
+        run: node --unhandled-rejections=strict ./check-e2e-wallet-balance.ts
```

It should use `ts-node` otherwise it could fail on TS syntax, some tweaks may be needed though, like using `import` instead of `require` to get types.
```suggestion
        run: yarn ts-node --unhandled-rejections=strict ./check-e2e-wallet-balance.ts
```

---

Fixed in https://github.com/valora-inc/wallet/pull/2325/commits/876bcbf710ddca64aa4a435021cd45ffbab09303 and https://github.com/valora-inc/wallet/pull/2325/commits/8ef142ea9af39f48f34bef82c345e71fb12d9de6!